### PR TITLE
[AR-29] Created HeaderInfoWithTabs for About Us Page

### PR DESCRIPTION
**Pull request by alex-baltz-17**
Capture

### CODE REVIEW

File: app/_components/Content/HeaderInfoWithTabs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import Image from "next/image";
+import React, { useState } from "react";
+import styled from "styled-components";
+
+const HeaderInfoWithTabsWrapper = styled.div`
```
Comment (idiglove) [inline app/_components/Content/HeaderInfoWithTabs/index.tsx]: hi sorry if I didnt make it clear, but we are going to use tailwind for new components now. can you refactor this? :)

great effort tho and thanks for the screenshot

**Review by idiglove — COMMENTED** [review]
i have another thing, this will be a running PR which means, we can create more PRs to point to this.

The aim of this PR is to create a new page under app folder

I prefer to not use pages folder with this now

That means - the about page in this PR will only have the Header.
What you can do too is add a FeatureFlag,
1- Add a new page in app folder named about-new.tsx
2 - only render this if the process.env.NEXT_ABOUT_NEW=true

lemme know if have questions

File: app/_components/Content/HeaderInfoWithTabs/index.tsx (hunk 1 above)
Comment (alex-baltz-17) [reply app/_components/Content/HeaderInfoWithTabs/index.tsx]: Sure, I will refactor the styling now. Thank you!

**Comment by alex-baltz-17** [comment]
Done refactoring.
Changes:
Undo changes to about.tsx then renamed with prefix underscore to disable
Undo changes to components/Header/index.tsx
Undo changes to components/Header/styled.tsx
Created about-new page
Created HeaderInfoWith tabs as reusable component
Implemented FeaturedFlag as per comment above

File: app/about-new/page.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import HeaderInfoWithTabs from "app/_components/Content/HeaderInfoWithTabs";
+import React from "react";
+
+const About = () => {
+  return (
```
Comment (idiglove) [inline app/about-new/page.tsx:9]: or you can do before the return function

```
if (process.env.NEXT_PUBLIC_ABOUT_NEW === "true") {
return null
}

...
```

**Review by idiglove — APPROVED** [review]
Just another small comment, I'm approving it.
Good job and real quick! <3 

I will not merge this yet, and we can check out from this branch for next pages

**Pull request merged**